src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml

```diff
@@ -196,38 +196,49 @@ else
+                    <div class="row">
+                        <div class="contact_form clearfix">
+                            <div class="col-md-4 col-sm-6">
                                 <div class="text_field mc-field-group">
-                                    <input type="text" placeholder="Imię" value="" name="FNAME" class="" id="mce-FNAME">
+                                    <input type="email" placeholder="E-mail" value="" name="email" class="required email" id="mce-EMAIL">
                                 </div>
                             </div>
 
-                            <div class="col-md-3 col-sm-6">
+                            <div class="col-md-4 col-sm-6">
                                 <div class="text_field mc-field-group">
-                                    <input type="text" placeholder="Nazwisko" value="" name="LNAME" class="" id="mce-LNAME">
+                                    <input type="text" placeholder="Imię i nazwisko" value="" name="name" class="" id="mce-NAME">
                                 </div>
                             </div>
 
-                            <div class="col-md-3 col-sm-6">
+                            <div class="col-md-4 col-sm-6">
                                 <div id="mce-responses" class="clear">
                                     <div class="response" id="mce-error-response" style="display: none"></div>
                                     <div class="response" id="mce-success-response" style="display: none"></div>
                                 </div> <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
                                 <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_8273a841f65d9cbb0055b7672_76610be8a5" tabindex="-1" value=""></div>
                                 <div class="clear content-centered"><input type="submit" value="Zapisz się" name="subscribe" id="mc-embedded-subscribe" class="chris_btn reveal animated" data-reveal-anim="fadeIn"></div>
                             </div>
-                        </form>
+                        </div>
                     </div>
-                </div>
+                    <div class="row">
+                        <div class="col-md-12 col-sm-12">
+                            <div class="mc-field-group input-group checkbox">
+                                <label>
+                                    <input type="checkbox" name="gdpr" id="mce-GDPR" class="required">
+                                    <strong>Zgoda na przetwarzanie danych</strong>: Zostawiając tutaj swojego maila zgadzasz się na postanowienia regulaminu dostępnego tutaj:
+                                    <a href="https://2018.devadventcalendar.pl/regulamin.pdf">Regulamin.pdf</a>
+                                </label>
```

Comment:
please do not use https://2018.devadventcalendar.pl/regulamin.pdf - use https://devadventcalendar.pl/regulamin.pdf (add if it does not exist)